Translated Description

DahliaSR opened the pull request.

DahliaSR committed:
Translated Short Description

Part of #1

DahliaSR committed:
Translated Description

Closes #1

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -59,69 +57,69 @@
-    Stephan would lead this mission (yeah, that probably was a bad idea).
-    He forgot how to do multiplication and tried to invent a new method.
-    It’s a rather strange method if we may be so blunt.
+    Nach der Lektüre von "Alice im Wunderland ", haben sich unsere Roboter entschieden,
```
Are both commas here necessary? It's so bumby 🐫I'd dare to go with none. Even the original should at least take the comma out the quotation marks.

DahliaSR replied:
The firt one is wrong, the second is optional...
Let's get rid of them.

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -59,69 +57,69 @@
-    He forgot how to do multiplication and tried to invent a new method.
-    It’s a rather strange method if we may be so blunt.
+    Nach der Lektüre von "Alice im Wunderland ", haben sich unsere Roboter entschieden,
+    ihr eigenes Einmaleins zu erstellen. Stephan wollte diese Mission leiten
```
He probably didn't want to. would is the past tense of will.

DahliaSR replied:
Yea, BUT Bryukh is the author of the task!
And it matches the context.

GiForce replied:
maybe a missunderstanding? 🐙 _would lead this mission_ wie Bryukh es formuliert, heisst nicht _wollte diese Mission leiten_ sondern _sollte diese Mission leiten_  oder _hätte ..sollen_. would != wollen. 
_Würde_ könnte man auch übersetzen, passt aber weniger.

DahliaSR replied:
_would lead this mission_ and _was a bad idea_
**was** ist Vergangenheit, man kann also annehmen das Stephan die Leitung bereits übernommen hat. Schliesslich besteht die Aufgabe ja darin Stephans Variante umzusetzten,
was wohl nicht nötig wäre, wenn Stephan nicht zuständig ist.

Der Kontext ist leider nicht ganz klar, was vermutlich daran liegt, dass Englisch nicht die Muttersprache von Bryukh ist. Solche Unstimmigkeiten sind mir bei seinen Aufgaben schon öfter aufgefallen. Hier ist es also an uns beiden eine brauchbare Lösung zu finden.

Wir können, denke ich, zwei Wege gehen:  
- Stephan _hat_ die Leitung übernommen **und** es _war_ ein schlechte Idee
  **oder**
- Stephan _möchte_ die Leitung übernommen **und** es _ist_ eine schlechte Idee

Also ICH tendiere ja zu ersterem... :stuck_out_tongue_winking_eye:

DahliaSR replied:
:white_check_mark:

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -59,69 +57,69 @@
+    ihr eigenes Einmaleins zu erstellen. Stephan wollte diese Mission leiten
+    (ja, das war wahrscheinlich eine schlechte Idee). Er hat allerdings vergessen wie man
+    multipliziert und dann versucht eine neue Methode zu erfinden.<br>
+    Es ist eine ziemlich seltsame Methode, wenn wir so direkt sein wollen.
```
I would change that last sentence.
Just "Eine ziemlich seltsame Methode, wenn wir ehrlich sein wollen.." sounds better
or even "Eine ziemlich seltsame Methode, unverblümt( od freiheraus, od ehrlich) gesagt"

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -343,40 +335,40 @@
-    You should help Stephan write a function to calculate this "multiplication".
-    You are given two positive integers (n&nbsp;>&nbsp;0), be careful with order of arguments.
+    Du sollst Stephan helfen eine Funktion zu schreiben, die diese Art von "Multiplikation"
+    anwenden kann.
```
we say eine Funktion _führt aus_,  rather than _wendet an_, dh: _..die diese Art von  "Multiplikation" ausführen kann_

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -343,40 +335,40 @@
-
+    <strong>Wozu das gut ist:</strong><br>
+    In dieser Aufgabe spielen wir mit logischen binären Operationen - der Grundlage der
+    Informatik - rum.<br>
```
That ending, please not. End after Informatik. That's so to emphasize, you could even have an exclamation mark there.

GiForce commented on translations/de/info/task_short_description.html:
```diff
@@ -1 +1,2 @@
-"Alice's Adventures in Wonderland" inspired our robots to create their own Multiplication Table. One little problem: Stephan forgot how to do multiplication…
\ No newline at end of file
+"Alice im Wunderland" hat unsere Roboter inspiriert ihr eigenes Einmaleins zu erstellen.<br>
+Es gibt allerdings ein kleines Problem: Stephan vergessen, wie man multipliziert...
```
"E. T. nach Hause telefonieren".  ;)

DahliaSR replied:
"Wie multiplizieren man, vergessen Stephan hat" - Meister Yodaß!?! :sunglasses:

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -59,69 +57,69 @@
+    ohne führende Nullen konvertiert. Dann wird die erste Zahl vertikal geschrieben
+    (von oben nach unten) und die Zweite horizontal (von links nach rechts).
+    Damit füllen wir je eine Wertetabelle für die binären Operationen AND, OR und XOR,
+    sodass wir am Ende drei Tabellen haben.
```
Your describtion of the notation is unclear and actually wrong.

> Damit füllen wir je....

We don't fill the table with the previous mentioned vertic- and horizontally written numbers. The results of the binary operations between the column values (horizontally number) and the row values fill these tables. Like the original states, where they cross each other tablewise.

GiForce reviewed (changes requested):
Don't blame Stephan ;)

DahliaSR committed:
Fixes according to review

https://github.com/CheckIO-to-German/Multiplication-Table/pull/2#pullrequestreview-3444593

DahliaSR committed:
Fixes according to review

https://github.com/CheckIO-to-German/Multiplication-Table/pull/2#pullrequestreview-3444593

GiForce commented:
Yes, yes, Yoda and E.T., buddies they are 

> Am 10.10.2016 um 20:57 schrieb DahliaSR notifications@github.com:
> 
> @DahliaSR commented on this pull request.
> 
> In translations/de/info/task_short_description.html https://github.com/CheckIO-to-German/Multiplication-Table/pull/2:
> 
> > \ No newline at end of file
> > +"Alice im Wunderland" hat unsere Roboter inspiriert ihr eigenes Einmaleins zu erstellen.<br>
> > +Es gibt allerdings ein kleines Problem: Stephan vergessen, wie man multipliziert...
> > "Wie multiplizieren man, vergessen Stephan hat" - Meister Yodaß!?! 😎
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub https://github.com/CheckIO-to-German/Multiplication-Table/pull/2, or mute the thread https://github.com/notifications/unsubscribe-auth/ARBj_YiZXOHQvc3OpaCc_g9ew1qw3nebks5qyoqggaJpZM4KSF-0.

DahliaSR committed:
Merge branch '#1-Translate-Description' of https://github.com/CheckIO-to-German/Multiplication-Table into #1-Translate-Description

GiForce commented:
On track, we agree on the narrative tense!
Dachte, wenn es etwa heisst  „Stephan sollte diese Mission leiten (yeah, das war wahrscheinlich keine gute Idee)“ , dann bedeutet das,  Er leitete die Mission – bis er mit seiner komischen Methode daherkommt, und deswegen dann der geehrte checkIO - Coder ranmuss, also die Mission übernehmen (leiten) muss.
Aber vielleicht dieser Art: „Stephan hätte diese Mission leiten sollen. Aber das war vielleicht keine so gute Idee, denn er vergass wie man multipliziert und erfand dafür eine eigene Methode. Und die ist eher seltsam, wenn wir ehrlich sind"

> Am 11.10.2016 um 22:34 schrieb DahliaSR notifications@github.com:
> 
> @DahliaSR commented on this pull request.
> 
> In translations/de/info/task_description.html https://github.com/CheckIO-to-German/Multiplication-Table/pull/2:
> 
> > ```
> >  </p>
> > ```
> 
> ```
>  <p style="clear: both;"></p>
> ```
> 
>  </div>
> 
>  <p>
> -    After reading "Alice's Adventures in Wonderland,"
> -    our robots decided to create their own "Multiplication table."
> -    Stephan would lead this mission (yeah, that probably was a bad idea).
> -    He forgot how to do multiplication and tried to invent a new method.
> -    It’s a rather strange method if we may be so blunt.
> -    Nach der Lektüre von "Alice im Wunderland ", haben sich unsere Roboter entschieden,
> -    ihr eigenes Einmaleins zu erstellen. Stephan wollte diese Mission leiten
>   would lead this mission and was a bad idea
>   was ist Vergangenheit, man kann also annehmen das Stephan die Leitung bereits übernommen hat. Schliesslich besteht die Aufgabe ja darin Stephans Variante umzusetzten,
>   was wohl nicht nötig wäre, wenn Stephan nicht zuständig ist.
> 
> Der Kontext ist leider nicht ganz klar, was vermutlich daran liegt, dass Englisch nicht die Muttersprache von Bryukh ist. Solche Unstimmigkeiten sind mir bei seinen Aufgaben schon öfter aufgefallen. Hier ist es also an uns beiden eine brauchbare Lösung zu finden.
> 
> Wir können, denke ich, zwei Wege gehen:
> 
> Stephan hat die Leitung übernommen und es war ein schlechte Idee oder
> Stephan möchte die Leitung übernommen und es ist eine schlechte Idee
> Also ICH tendiere ja zu ersterem... 😜
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub https://github.com/CheckIO-to-German/Multiplication-Table/pull/2, or mute the thread https://github.com/notifications/unsubscribe-auth/ARBj_fzwA2EsPhJ0ZljrsKfoyfi8PIOCks5qy_LAgaJpZM4KSF-0.

DahliaSR commented:
> Aber vielleicht dieser Art: „Stephan hätte diese Mission leiten sollen. Aber das war vielleicht keine so gute Idee, denn er vergass wie man multipliziert und erfand dafür eine eigene Methode. Und die ist eher seltsam, wenn wir ehrlich sind"

Das ist Super... naja fast!
- _Aber das war vielleicht keine_ ==> _Das war vermutlich keine_ **oder** _Das war keine_
- _denn er vergass wie man_ ==> klingt nach Märchenstunde _denn er hat vergessen wie man_

Passt das so?

GiForce commented:
🤖💬✅

> On 12 Oct 2016, at 21:23, DahliaSR notifications@github.com wrote:
> 
> Aber vielleicht dieser Art: „Stephan hätte diese Mission leiten sollen. Aber das war vielleicht keine so gute Idee, denn er vergass wie man multipliziert und erfand dafür eine eigene Methode. Und die ist eher seltsam, wenn wir ehrlich sind"
> 
> Das ist Super... naja fast!
> 
> Aber das war vielleicht keine ==> Das war vermutlich keine oder Das war keine
> denn er vergass wie man ==> klingt nach Märchenstunde denn er hat vergessen wie man
> Passt das so?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub, or mute the thread.

DahliaSR committed:
More Fixes according to review

https://github.com/CheckIO-to-German/Multiplication-Table/pull/2#pullrequestreview-3444593

GiForce reviewed (approved):
🎃

DarkoRevan reviewed (approved):
Nice!
